### ISSUE TITLE
Contract test - additional properties

### ISSUE THREAD

**Issue by thim81** [apideck-libraries/portman#192]
"additionalProperties: false" is a powerful OpenAPI property, since it allows you to specify if your API has an evolving response or fixed response, when it comes to the properties documented.

This feature would allow Portman to inject/overwrite the "additionalProperties" setting for the response object, which is using in the Schema validation of a response.

Proposal:
```
     {
        "openApiOperation": "*::/crm/*",
        "schemaValidation": {
          "enabled": true,
          "additionalProperties": false
        }
      }
```

**Comment by savage-alex** [comment]
I would like this and should have raised a request here shouldn't I ;-)

**Comment by Gdewilde** [comment]
Thanks for the suggestion, @savage-alex!

**Comment by nicklloyd** [comment]
And yes @savage-alex - this is the place ;)

**Comment by thim81** [comment]
@savage-alex We have a PR in the making for providing the contract test extension for the JSON Schema validation.

Depending on the review & feedback, this will refined to make it part of one of the upcoming releases.

**Comment by savage-alex** [comment]
@AliKhalili is our Portman Expert. I will review with him. thanks all!

**Comment by savage-alex** [comment]
@thim81 I have tested it using the above setting and its not picking up additionalProperties I have included. Did you want a short call to discuss?

**Comment by thim81** [comment]
@savage-alex How did you test it? 

We only just released the feature in Portman 1.13, so if you were testing it with the previous version, it is most-likely that the `additionalProperties` was not accepted by Portman.

**Comment by savage-alex** [comment]
hey I tested it by de-bugging the PR. I can see the additionalProperties is injected in postman just fine:  
![image](https://user-images.githubusercontent.com/25430683/146939676-e560ffc2-707b-43e7-a477-6938a5966734.png)
What I cannot work out is why it is flagging a response as having additional properties when it matches the schema

**Comment by savage-alex** [comment]
Ignore me. It was an actual additional property!! Works great

**Comment by thim81** [comment]
@savage-alex Are you OK, that we close the issue?

**Comment by savage-alex** [comment]
Please do.

**Comment by thim81** [comment]
Closing the issue, since the feature has landed in Portman version 1.13.
